src/main/java/org/opensearch/performanceanalyzer/rca/store/rca/admissioncontrol/heap/HeapRcaFactory.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package org.opensearch.performanceanalyzer.rca.store.rca.admissioncontrol.heap;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** HeapRcaFactory returns HeapRca based on maxHeap in gigabytes */
+public class HeapRcaFactory {
+
+    // Heap Size in Gigabytes
+    private static final int SMALL_HEAP_RCA_THRESHOLD = 4;
+    private static final int MEDIUM_HEAP_RCA_THRESHOLD = 32;
+
+    // Keys for rcaMap
+    private static final String small = "small";
+    private static final String medium = "medium";
+    private static final String large = "large";
```

Comment:
static final variables names usually follow upper cases. Also can we be more descriptive with the variable name? ```SMALL_HEAP```?